web/scripts/app.js

```diff
@@ -230,7 +230,9 @@ function search() {
     var rowid;
     if (query === searchInfo.laststr) {
         rowid = searchInfo.lastrow;
+        searchbutton.textContent = "Next";
```

Comment:
It would be `searchbutton.innerHTML = 'Next'`. But maybe we don't actually need a UI hint, and can just always leave the text as 'Search'.